src/main/java/fi/aalto/cs/apluscourses/intellij/notifications/CourseFileError.java

```diff
@@ -0,0 +1,23 @@
+package fi.aalto.cs.apluscourses.intellij.notifications;
+
+import static fi.aalto.cs.apluscourses.utils.PluginResourceBundle.getAndReplaceText;
+import static fi.aalto.cs.apluscourses.utils.PluginResourceBundle.getText;
+
+import com.intellij.notification.Notification;
+import com.intellij.notification.NotificationType;
+import fi.aalto.cs.apluscourses.intellij.services.PluginSettings;
+
+public class CourseFileError extends Notification {
+
+  /**
+   * Error to be shown when course settings file cannot be accessed.
+   *
+   * @param e Exception.
+   */
+  public CourseFileError(Exception e) {
+    super(PluginSettings.A_PLUS,
+        getText("notification.CourseFileError.title"),
+        getAndReplaceText("notification.CourseFileError.content", e.getLocalizedMessage()),
```

Comment:
I'm curious if there's any specific reason to use `getLocalizedMessage` here? Hasn't been used elsewhere as far as I know

Reply:
No reason. I somehow assumed/remembered we used localized messages in the UI. Changed that.